app/views/tran_details/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<p style="color: green"></p>
+<h2>GROUP TRANSACTIONS</h2>
+<% @group_name = TranGroup.find(params[:tran_group_id]) %>
+<h5><strong>Group Name: </strong> <%= @group_name.groupName %></h5>
+<h5 style="color:skyblue"><strong>Group Total :</strong> <%= @total_value %></h5>
+<div id="tran_details">
+  <table class="table table-hover">
+    <thead>
+      <tr>
+        <th scope="col">Date</th>
+        <th scope="col">Transection Details</th>
+        <th scope="col">Amount</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @tran_details.each do |tran|%>
+        <tr>
+          <td scope="row"><%= tran.created_at.strftime('%d-%m-%y')%></td>
+          <td scope="row"><%= tran.itemDetails %></td>
+          <td scope="row"><%= tran.amount %></td>
```

Comment:
- [ ] You'll equally required the currency symbol as in the provided design guidelines :+1:

Reply:
resolved